src/pages/buttons/demo/basic.html

```diff
@@ -0,0 +1,50 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>Buttons</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content padding>
+  <p>
+    <button ion-button>Default</button>
+  </p>
+  
+  <p>
+    <button ion-button color="secondary">Color</button>
+  </p>
+
+  <p>
+    <button ion-button color="secondary" outline>Outlined</button>
```

Comment:
Can we remove the secondary color here since we're only focusing on the outline property?